### PR TITLE
J unit

### PR DESCRIPTION
**Pull request by gmit22**
Addresses #192

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,7 +27,14 @@ test {
         showStandardStreams = false
     }
 }
-
+task Checkstyle(type: Checkstyle) {
```
Comment (keke101) [inline build.gradle]: Since we already have a checkstyle gradle task which are the `checkstyleMain` and `checkstyleTest`, is this the same as those 2?

Comment (gmit22) [reply build.gradle]: Yes it is, my checkstyle was not working for some reason so I had to this manually.

Comment (keke101) [reply build.gradle]: Did you try re-cloning the project?

Comment (gmit22) [reply build.gradle]: Like deleting it and initializing it again?
I did not do that

Comment (keke101) [reply build.gradle]: As in you don't have to delete the current folder you are working on, just clone your fork in a separate folder and open it with intellij. Then test the checkstyle there. If it works, you can remove this section.

**Review by keke101 — APPROVED** [review]

**Pull request merged**